### ISSUE TITLE
Arrows get stuck in FloatingTextParticle

### ISSUE THREAD

**Issue by AndreyNazarchuk** [iTXTech/Genisys#2045]
Arrows are getting stuck in floating text.

To reproduce just shoot some arrows just below the text, they will get stuck and vibrate there.

Evidence:
![screenshot_20161008-212446](https://cloud.githubusercontent.com/assets/7192775/19217913/02d3cb40-8d9e-11e6-8f62-cb0fbf864b1e.png)

**Comment by ABAxStorm** [comment]
Is it really that hard to follow the template?

**Comment by ishitatsuyuki** [comment]
## Improperly Formatted Issue Ticket - Lacks Information

Without knowing what your Version, OS, PHP, Client, or plugins and how the issue occurred, the issue cannot begin to be solved.
A friendly pre-formatted template is provided that shows the required information to begin to find, test and solve an issue.
Until the adequate template is provided, the issue will be closed; you are fully welcome to update the issue ticket with the adequate information and we will gladly reopen the issue.
Sorry for the inconvenience.

**Comment by AndreyNazarchuk** [comment]
People would report way more bugs if you guys weren't so anal about how they should do it.

This bug was found on PHP 7, latest build of Genisis, Ubuntu 16.04

**Comment by ABAxStorm** [comment]
Well they can't fix them if they don't know where or how it happened.

**Comment by dktapps** [comment]
Well, actually I know exactly why this would happen. Highly amusing actually.

**Comment by Tarik02** [comment]
This is because of implementation of FloatingTextParticle: when you show text particle, you will spawn player with name as text of particle. If you shoot to particle, you shoot to player, so arrows will affect invisible player.

**Comment by legoboy0215** [comment]
The FTP is actually an Entity.

**Comment by XFuryMCPE** [comment]
How about we make arrows not stop when colliding with invisible players? Would that work?

**Comment by Tarik02** [comment]
@XFuryMCPE, this is client problem. Possible fix of this is broadcasting arrow movement, but this is not good way to fix this bug.

**Comment by dktapps** [comment]
This has been fixed on a development branch, the fix will be merged to master in the future (no ETA can be provided).
